### PR TITLE
Add dls-controls as provider

### PR DESCRIPTION
**Pull request by stan-dot**
next step: add `catalog-info.yml` to the repos

### CODE REVIEW

**Comment by coretl** [comment]
dls-controls is going away. We need to make that happen soon

**Comment by garryod** [comment]
> dls-controls is going away. We need to make that happen soon

Would you prefer we didn't add it?

**Comment by stan-dot** [comment]
adding it as a source is quick, if the 'moving away from dls-controls' is even as little as week away it's no harm to migrate it now and then back. let's live in the present

File: backstage/app-config.yaml [hunk 1]
```diff
@@ -96,6 +96,14 @@ catalog:
             minutes: 60
           timeout:
             minutes: 10
+      GitHub-DLS-Controls:
+        organization: 'dls-controlst '
```
Comment (garryod) [inline backstage/app-config.yaml:100]:
```suggestion
        organization: 'dls-controls'
```

**Comment by garryod** [comment]
Closed in favour of migrating from `dls-controls` to `DiamondLightSource`

**Pull request closed without merging**